src/pages/Chat/components/ChatRoomsList/ChatRoomsListItem.tsx

```diff
@@ -0,0 +1,54 @@
+import {
+  Block,
+  LastMessage,
+  ChatElement,
+} from "pages/Chat/components/ChatRoomsList/ChatRoomsList.styles";
+import { Avatar } from "@mui/material";
```

Comment:
Can you change this to the antd avatar variation? We are using it on other components

Reply:
done